### PR TITLE
Add polyglot template Helper from the userPage branch.

### PR DESCRIPTION
**Pull request by jjeffryes**

### CODE REVIEW

File: js/utils/templateHelpers.js [hunk 1]
```diff
@@ -1,5 +1,10 @@
-// todo: add in some helpers!
+import app from '../app';
+
+const polyT = function (...args) {
+  return app.polyglot.t(...args);
+};
+
+export {
```
Comment (rmisio) [inline js/utils/templateHelpers.js]: Can we directly export the polyT function?

```
export function plyT(...args) {
  return app.polyglot.t(...args);
}
```

In ES6 modules, you can export multiple things individually, for example:

```
export function a() {};

export function b() {};

const variableA = 'meatballs';

export { variableA };
```

The benefit is that then individual items can be imported and only those would go into memory. If I'm not mistaken, if you just export everything in one big object, I think the whole thing gets brought in.

Comment (jjeffryes) [inline js/utils/templateHelpers.js]: LOL. That's how I had it before, I changed it to this based on our conversation earlier about switching to this pattern.

File: js/utils/templateHelpers.js [hunk 2]
```diff
@@ -1,10 +1,6 @@
...
 
-const polyT = function (...args) {
-  return app.polyglot.t(...args);
-};
-
-export {
-  polyT,
-};
+export function polyT(...args) {
+  return app.polyglot.t(args);
```
Comment (rmisio) [inline js/utils/templateHelpers.js]: I think you need the ellipses for the args your passing to app.polyglot.t as well.

File: js/utils/templateHelpers.js (hunk 1 above)
Comment (rmisio) [inline js/utils/templateHelpers.js]: well, we must have been playing 'whos on first?'. What's new, right?

File: js/utils/templateHelpers.js (hunk 2 above)
Comment (jjeffryes) [inline js/utils/templateHelpers.js]: Rolled back to the wrong state. I'll update it.

**Pull request merged**